RecoTracker/FinalTrackSelectors/python/earlyGeneralTracks_cfi.py

```diff
@@ -101,3 +101,24 @@
     makeReKeyedSeeds = cms.untracked.bool(False)
     )
 )
+from Configuration.ProcessModifiers.vectorHits_cff import vectorHits
+
+(trackingPhase2PU140 & vectorHits).toModify(earlyGeneralTracks, TrackProducers = ['initialStepTracks',
+										  'highPtTripletStepTracks',
+										  'lowPtQuadStepTracks',
+										  'lowPtTripletStepTracks',
+										  'detachedQuadStepTracks',
+										  'pixelPairStepTracks',
+									          'pixelLessStepTracks'],
+							 	hasSelector = [1,1,1,1,1,1,1],
+								indivShareFrac = [1.0,0.16,0.095,0.09,0.09,0.09,0.095],
+								selectedTrackQuals = ['initialStepSelector:initialStep',
+							                              'highPtTripletStepSelector:highPtTripletStep',
+                          							      'lowPtQuadStepSelector:lowPtQuadStep',
+                          							      'lowPtTripletStepSelector:lowPtTripletStep',
+                          							      'detachedQuadStep',
+                          							      'pixelPairStepSelector:pixelPairStep',
+										      'pixelLessStepSelector:pixelLessStep'],
+								setsToMerge = cms.VPSet( cms.PSet( tLists=cms.vint32(0,1,2,3,4,5,6), pQual=cms.bool(True)))
+)
+
```

Comment:
```suggestion
def _extend_pixelLess(x):
    x.TrackProducers += 'pixelLessStepTracks'
    x.hasSelector += 1
    x.indivShareFrac += 0.095
    x.selectedTrackQuals += 'pixelLessStepSelector:pixelLessStep'
    x.setsToMerge[0].tLists += 6
(trackingPhase2PU140 & vectorHits).toModify(earlyGeneralTracks, _extend_pixelLess)
```

as a shorter variant

Reply:
Done. However, I had some problems with the `x.setsToMerge[0].tLists += 6`, where python complains that an `int` object is not iterable. I have left this part written out expicitly for now.

Reply:
I think it should be `x.setsToMerge[0].tLists += [6]`

Reply:
strange, I made my suggestion based on a local test that showed no difference in the expanded config

Reply:
ah, nm, I thought 23234.9 was supposed to use vectorHits, but it doesn't

Reply:
@kpedro88 Ah, yes, that works. I thought I had tested that, as for the other elements, but apparently not. Works now, thanks.

Reply:
indeed, `[]` makes more sense. I interactively tested only the string case and only single-character `+= 'a'`, which actually does the right thing.
For the rest, I got trapped by the missing `vectorHits` in the test workflow 23234.9.
Sorry